libc/src/stdio/printf_core/parser.h

```diff
@@ -10,6 +10,7 @@
 #define LLVM_LIBC_SRC_STDIO_PRINTF_CORE_PARSER_H
 
 #include "include/llvm-libc-macros/stdfix-macros.h"
+#include "src/__support/CPP/algorithm.h" // max
```

Comment:
Can you also add `libc.src.__support.CPP.algorithm` to the `DEPENDS` list of `parser` target in the `CMakeLists.txt`, and `:__support_cpp_algorithm` to the `deps` list of `printf_parser` target in `llvm-project/utils/bazel/llvm-project-overlay/libc/BUILD.bazel`?  Thanks,

Reply:
Thanks @lntue - done.